release/update-rsync.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/sh -eu -o pipefail
```

Comment:
This is an out-of-scope change that affects the entire script, not just the directory-related code. The change itself is controversial: Many shell developers [believe](http://mywiki.wooledge.org/BashFAQ/105) "set -e" should not be used for error detection/handling. This addition breaks this script in my tests on Ubuntu (v20.04 and v22.04):

```
/bin/sh: 0: Illegal option - 
```

Please do not make this change it in this PR.

Reply:
done